### ISSUE TITLE
Read current temperature

### ISSUE THREAD

**Issue by kopr12** [faf0/sct#4]
Is there a way to do it maybe ?

**Comment by ilioni** [comment]
`xrandr --verbose |grep -i gamma`

**Comment by kopr12** [comment]
Yeah, though `xrandr --verbose |grep -i gamma:` is better.
Was thinking more of reading the value that's set, like `xsct 5400`, reading the actual 5400 value.

**Comment by zvezdochiot** [comment]
@kajzersoze say> reading the actual... value.

Use ```awk```:
```
xrandr --verbose | grep -i gamma | sed -e 's/inf/100000000/g' | awk -F: '{ print 1.0 " " $2/$3 " " $2/$4 }' | awk '{ print 64465-109049*$1+46013*$1*$1-4322*$2+10708*$2*$2-2662*$3+1355*$3*$3 }'
```

**Comment by kopr12** [comment]
I've tested your fix and it works, that's what I needed, thanks!

**Comment by kopr12** [comment]
Worked on the first run, but if you run it again it seems to set and read values incorrectly.

![20190804-220136](https://user-images.githubusercontent.com/11412353/62428618-bf132a80-b703-11e9-9411-e4b57619d11a.png)

**Comment by zvezdochiot** [comment]
The temperature value is not readable (not from where to read). It is approximated (estimated) by gamma.

**Comment by kopr12** [comment]
I get that, but trust me, it was setting the gamma wrong, that one in the picture with 3400, I'm using 3400 for a very long time, this time it was darker, perhaps the reading was right then, I guess it was set to 3000 instead of 3400

**Comment by zvezdochiot** [comment]
I generally wanted to replace the table with approximation. But the author refers to `redshift`. Why drop out of trend?

**Comment by kopr12** [comment]
Oh I see (I just looked at the code in PR, didn't before) table is still in there you're just not using it, table is actually longer in redshift, author trimmed it down, well ok, interesting approach, I'll test it more.

**Comment by zvezdochiot** [comment]
To evaluate the accuracy of the approximation:
```
$ ./xsct 
Screen 0: temperature ~ 3107

$ redshift -p
Time: Night
Color Temperature: 3500K
Brightness: 1.00
```

**Comment by kopr12** [comment]
Actually you are using that table, just in a different way.
Problem I have with this fix is in steps, although the table used 500 steps I was able to see a difference when setting 100 steps, with fix this is not a case anymore.
That's why I saw the difference when setting 3400 before and after the fix.
Setting the value for example in range 3500 - 3900 is the same.
I don't see why the table from redshift had to become smaller, using 100 steps is better.

![20190805-011620](https://user-images.githubusercontent.com/11412353/62430672-eb3ca480-b71f-11e9-929c-b8ed67a0f97a.png)
